xwiki-platform-distribution/xwiki-platform-distribution-debian/pom.xml

```diff
@@ -69,6 +69,47 @@
     </extensions>
     <pluginManagement>
       <plugins>
+        <plugin>
+          <!-- Get the current year -->
+          <groupId>org.codehaus.mojo</groupId>
+          <artifactId>build-helper-maven-plugin</artifactId>
+          <executions>
+              <execution>
+                  <id>timestamp-property</id>
+                  <goals>
+                      <goal>timestamp-property</goal>
+                  </goals>
+                  <phase>validate</phase>
+                  <configuration>
+                      <name>current.year</name>
+                      <pattern>yyyy</pattern>
+                  </configuration>
+              </execution>
+          </executions>
+        </plugin>
+        <plugin>
+            <groupId>org.xwiki.commons</groupId>
+            <artifactId>xwiki-commons-tool-remote-resource-plugin</artifactId>
+            <executions>
+                <execution>
+                    <id>xwiki-platform-tool-configuration-resources</id>
+                    <phase>process-resources</phase>
+
+                    <configuration>
+                        <resourceBundles>
+                            <!-- Using ${platform.version} on purpose here. An XWiki distribution with a different version might inherit this. -->
+                            <resourceBundle>org.xwiki.platform:xwiki-platform-tool-configuration-resources:${platform.version}</resourceBundle>
```

Comment:
Why xwiki-platform-tool-configuration-resources ?

Reply:
It's defined in the parent pom but overridden if I specify a new resourceBundle here, so I need to specify them both. This one is used for generating the hibernate.cfg and other config files.

Reply:
it looks pretty obvious now you tell me... thanks!

Reply:
Finally it's not working: if I rename this one, only the parent one is executed, and this one remains overridden. Apparently it's not linked to the name: tried to rename them xwiki-1 and xwiki-2 and only "xwiki-1" appears in my console.